### DISCUSSION TITLE
NBMiner v41.5

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#873] [General]
- `feature`: `ethash` Add 90% LHR unlocker for `3080 12G` & `3050`.
- `feature`: `ergo` Add LHR unlocker support.
- `fix`: `ethash` LHR unlocker improve stability.

<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v41.5'>NBMiner v41.5</a>.</em>

**Comment by Oleg1606** [comment]
Только у меня 100% загрузка процессора?

**Comment by Danishagn** [comment]
3060 V2
TRIED ON EVERY VERSION
41.1, 41.2, 41.3, 41.4, 41.5
0% fan and hashrate drop 
randomly cards 
510.68.02, 510.60.02 hiveos latest version

please look into this

Using OC
Core 1500, memory 2100 to 2000 
47 mhs to 46 mhs 

this happen with t-rex miner also

**Reply by nnobrecripto** [reply]
needing some help. I bought an asus dual rtx 3060 12gb v2 and wanted to know if it is compatible. I only watch video of other models. And if possible, any overclocking recommendations.

**Reply by scoverstudio** [reply]
I got same problem here.

**Reply by nnobrecripto** [reply]
I put this config in The Last V nbminer and i think is good
![20220521_062543](https://user-images.githubusercontent.com/105960880/169645194-b3358734-418c-49ee-acd5-5eef497237c3.jpg)

**Reply by nnobrecripto** [reply]
I used gpu tweak 2 for config, gpu boost: user def: linear 1200. Next try don't up a lot the mm clock, fan 80% and power 85%, tempo 70⁰

**Comment by nnobrecripto** [comment]
needing some help. I bought an asus dual rtx 3060 12gb v2 and wanted to know if it is compatible. I only watch video of other models. And if possible, any overclocking recommendations.
